.devcontainer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM eliteencoder/comfyui-base:latest
```

Comment:
Can we have this under `livepeer` org? Maybe `livepeer/comfystream-base`?

Reply:
I'll update this in the next push. @victorges How did we solve for the large container size limitation for GitHub actions in ai-worker? It would be nice to have a github action workflow to push to livepeer dockerhub.

Reply:
I'm not really sure. Maybe @hjpotter92 should help.

One potential issue is that we might not have access to the same environment outside of `livepeer` org? If that's the case we may need to fork it or setup the CI on `ai-worker` repo instead, which would be relatively confusing.